### PR TITLE
update debug config

### PR DESCRIPTION
**Pull request by liulianci**

### CODE REVIEW

File: ern-container-gen-ios/src/hull/Config/ElectrodeContainer-Debug.xcconfig [hunk 1]
```diff
@@ -5,11 +5,8 @@
 // https://github.com/dempseyatgithub/BuildSettingExtractor
 //
 
-ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES
 CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES = YES
-CLANG_ENABLE_MODULES = YES
-CLANG_WARN_DOCUMENTATION_COMMENTS = NO
-CODE_SIGN_IDENTITY = 
+CODE_SIGN_IDENTITY =
```
Comment (gmbharath12) [inline ern-container-gen-ios/src/hull/Config/ElectrodeContainer-Debug.xcconfig]: why is this necessary ?

Comment (liulianci) [reply ern-container-gen-ios/src/hull/Config/ElectrodeContainer-Debug.xcconfig]: these configurations cause the container compiling issues

Comment (gmbharath12) [reply ern-container-gen-ios/src/hull/Config/ElectrodeContainer-Debug.xcconfig]: These are not required. Working for me on Debug config without these changes.

Comment (liulianci) [reply ern-container-gen-ios/src/hull/Config/ElectrodeContainer-Debug.xcconfig]: are you able to build the debug container using new build system?

Comment (gmbharath12) [reply ern-container-gen-ios/src/hull/Config/ElectrodeContainer-Debug.xcconfig]: Yes, I could build the Debug config mode without these changes.
